pandas/core/window.py

```diff
@@ -821,23 +877,92 @@ class Rolling(_Rolling_and_Expanding):
 
     Parameters
     ----------
-    window : int
+    window: int
         Size of the moving window. This is the number of observations used for
         calculating the statistic.
-    min_periods : int, default None
+    min_periods: int, default None
         Minimum number of observations in window required to have a value
         (otherwise result is NA).
-    freq : string or DateOffset object, optional (default None) (DEPRECATED)
+    freq: string or DateOffset object, optional (default None) (DEPRECATED)
         Frequency to conform the data to before computing the statistic.
         Specified as a frequency string or DateOffset object.
-    center : boolean, default False
+    center: boolean, default False
         Set the labels at the center of the window.
-    axis : int, default 0
+    on: string, optional
+        For a DataFrame, column on which to calculate
+        the rolling window, rather than the index
+
+      .. versionadded:: 0.18.2
+
+    axis: int or string, default 0
 
     Returns
     -------
     a Window sub-classed for the particular operation
 
+    Examples
+    --------
+
+    >>> df = DataFrame({'B': [0, 1, 2, np.nan, 4]})
```

Comment:
Can you use `pd.` here in the docstring examples as well? (like you did in the whatsnew tutorial docs)